src/sentry/notifications/notifications/codeowners_auto_sync.py

```diff
@@ -25,9 +25,12 @@ def determine_recipients(self) -> Iterable[Team | User]:
     def reference(self) -> Model | None:
         return None
 
-    def get_notification_providers(self) -> Iterable[ExternalProviders]:
```

Comment:
@mgaeta Why remove this method? It's a lot easier to limit the providers this way than it is in the updated method of `get_participants`